ambry-clustermap/src/main/java/com.github.ambry.clustermap/AmbryReplica.java

```diff
@@ -95,14 +96,18 @@ void setSealedState(boolean isSealed) {
     this.isSealed = isSealed;
   }
 
+  void setStoppedState(boolean isStopped) {
```

Comment:
This method is not used currently but might be helpful in the future PR when we would like dynamically change replica state on datanode.